src/user/user.service.ts

```diff
@@ -0,0 +1,68 @@
+import { Injectable } from '@nestjs/common';
+import { PrismaService } from 'src/prisma/prisma.service';
+import { User, Prisma } from '@prisma/client';
+
+@Injectable()
+export class UserService {
+  constructor(private prisma: PrismaService) {}
+
+  async findOneUser(email: string): Promise<User | undefined> {
```

Comment:
normally I'd prefer a service to throw a "Not Found` error but let's skip it as this is more a dao than a service